### ISSUE TITLE
Change osm.org to openstreetmap.org in docs

### ISSUE THREAD

**Issue by jakehm** [Leaflet/Leaflet#5129]
In the docs the example is:
```
L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
  attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
}).addTo(map);
```
osm.org is just for convenience in typing it into your browser.  But his causes a problem when using ssl.  You actually need to use `{s}.tile.openstreetmap.org/{z}/{x}/{y}.png`.  So why not change it to that in the docs?

**Comment by IvanSanchez** [comment]
Because nobody has done that yet? Feel free to submit a pull request with the changes.

**Comment by yohanboniface** [comment]
I guess this was fixed by #5130
